openfisca_aotearoa/entities.py

```diff
@@ -46,4 +46,45 @@
     is_person = True,
     )
 
-entities = [Titled_Property, Person]
+Family = build_entity(
+    key = "family",
+    plural = "families",
+    label = u'Family',
+    doc = '''
+    A Family represents a collection of related persons.
+
+    Family entities are required for calculations across a number of entitlements including for example "Working for families" and "Paid Parental Leave"
+
+    A family can contain a number of roles, such as 'principal_caregiver', 'partner', 'dependant_child' & 'independant_child'.
+
+    For more information on entities, see: http://openfisca.org/doc/coding-the-legislation/50_entities.html
+    ''',
+    roles = [
+        {
+            'key': 'principal_caregiver',
+            'plural': 'principal_caregivers',
+            'label': u'Principal caregivers',
+            'doc': u'The one person who is the principal caregiver of a family.'
```

Comment:
Is there a citation for this? IIRC a child can have > 1 principal carers. I'll search my email for that citation.

Reply:
I think the work might be primary principal carer, but that also can be more than one person

Reply:
for the RaaP work , we landed on a variable called `applicant.isPrincipalCarerForProportion` based on work with MSD. This must be > 33% to get Working for Families - Family Tax Credit. It's one example of a child having more than one principal carer.

Citation for 33% =  http://www.ird.govt.nz/wff-tax-credits/understanding/shared-care/
https://www.ird.govt.nz/resources/c/c/cc246bfd-b75a-458e-8f2b-be9a02ae4ec3/qb1601.pdf
Page 1
```
(b)    they live apart from whoever meets the requirement in (a), and 
they have the child in their exclusive care for certain minimum 
periods (so there is shared care of the child)
```

I'm looking for whether this is policy or legislation.